README.md

```diff
@@ -4,8 +4,8 @@ A PPA repository for SecureDNA packages
 # Usage
 
 ```bash
-sudo curl -SsL -o /etc/apt/trusted.gpg.d/securedna.gpg https://securedna.github.io/ppa/deb/securedna.gpg
-sudo curl -SsL -o /etc/apt/sources.list.d/securedna.list https://securedna.github.io/ppa/deb/securedna.list
+sudo curl -SsL https://securedna.github.io/ppa/deb/securedna.gpg | sudo tee /usr/share/keyrings/securedna-keyring.gpg > /dev/null
+sudo echo "deb [signed-by=/usr/share/keyrings/securedna-keyring.gpg] https://securedna.github.io/ppa/deb ./" | sudo tee /etc/apt/sources.list.d/securedna.list > /dev/null
```

Comment:
This is basically what I suggested in DMs, *but* you need to take the `sudo` out from in front of `curl` and `echo`---running them as root is incorrect, because that's why we're sudoing `tee` instead.